### PR TITLE
[Admin] (CATS) Resolve issues - April 2019

### PR DESCRIPTION
**Pull request by kbrandl**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[eacf6ad](https://github.com/OfficeDev/office-js-docs-pr/pull/894/commits/eacf6ad8b4a4568d5145069fdbe9bcbd37622640)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-debugging.md) | :bulb:Suggestion | | [Details](#user-content-fc1f0b9d-6af9a5eb)
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-json-autogeneration.md) | :bulb:Suggestion | | [Details](#user-content-32630023-6af9a5eb)
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |

<a id="fc1f0b9d-6af9a5eb"></a>
### [docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-debugging.md)
 - **Line 24**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Use-the-command-line-tools-to-debug&quot;&gt;command line instructions&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named Use-the-command-line-tools-to-debug.
 - **Line 35**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#debug-in-excel-online-by-using-the-browser-developer-tools&quot;&gt;using the browser tools to debug Excel Online&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named debug-in-excel-online-by-using-the-browser-developer-tools.
 - **Line 142**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Sideload-your-add-in&quot;&gt;Sideload your add-in&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named Sideload-your-add-in.

<a id="32630023-6af9a5eb"></a>
### [docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-json-autogeneration.md)
 - **Line 14**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Types&quot;&gt;Types&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-json-autogeneration.md doesn&#39;t contain a bookmark named Types.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904142327233970-894%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vI1NiSGud4o0MMtDkHT4bwpBjlBc4BvNeJSsnKnMn4c%3D&st=2019-04-14T23%3A23%3A45Z&se=2019-05-15T23%3A28%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/powerpoint/powerpoint-add-ins.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ localization_priority: Priority
...
+- Use **content add-ins** to add dynamic HTML5 content to your presentations. For example, see the [LucidChart Diagrams for PowerPoint](https://appsource.microsoft.com/en-us/product/office/WA104380117) add-in, which you can use to inject an interactive diagram from LucidChart into your deck.
 
-- Use **task pane add-ins** to bring in reference information or insert data into the presentation via a service. For example, see the [Shutterstock Images](https://store.office.com/app.aspx?assetid=WA104380169&ui=en-US&rs=en-US&ad=US&clickedfilter=OfficeProductFilter%3APowerPoint&productgroup=PowerPoint&homprd=PowerPoint&sourcecorrid=950950b7-aa6c-4766-95fa-e75d37266c21&homappcat=Editor%2527s%2BPicks&homapppos=0&homchv=1&appredirect=false) add-in, which you can use to add professional photos to your presentation. 
+- Use **task pane add-ins** to bring in reference information or insert data into the presentation via a service. For example, see the [Pixton Comic Characters](https://appsource.microsoft.com/en-us/product/office/WA104380907) add-in, which you can use to add professional photos to your presentation.
```
Comment (ElizabethSamuel-MSFT) [inline docs/powerpoint/powerpoint-add-ins.md]: Recommend removing the locale from the link

File: docs/powerpoint/powerpoint-add-ins.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ localization_priority: Priority
 
 You can use PowerPoint add-ins to build engaging solutions for your users' presentations across platforms including Windows, iOS, Office Online, and Mac. You can create two types of PowerPoint add-ins:
 
-- Use **content add-ins** to add dynamic HTML5 content to your presentations. For example, see the [LucidChart Diagrams for PowerPoint](https://store.office.com/app.aspx?assetid=WA104380117&ui=en-US&rs=en-US&ad=US&clickedfilter=OfficeProductFilter%3APowerPoint&productgroup=PowerPoint&homprd=PowerPoint&sourcecorrid=950950b7-aa6c-4766-95fa-e75d37266c21&homappcat=Productivity&homapppos=3&homchv=2&appredirect=false) add-in, which you can use to inject an interactive diagram from LucidChart into your deck.
+- Use **content add-ins** to add dynamic HTML5 content to your presentations. For example, see the [LucidChart Diagrams for PowerPoint](https://appsource.microsoft.com/en-us/product/office/WA104380117) add-in, which you can use to inject an interactive diagram from LucidChart into your deck.
```
Comment (ElizabethSamuel-MSFT) [inline docs/powerpoint/powerpoint-add-ins.md]:
```suggestion
- Use **content add-ins** to add dynamic HTML5 content to your presentations. For example, see the [LucidChart Diagrams for PowerPoint](https://appsource.microsoft.com/product/office/WA104380117) add-in, which you can use to inject an interactive diagram from LucidChart into your deck.
```

File: docs/powerpoint/powerpoint-add-ins.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [inline docs/powerpoint/powerpoint-add-ins.md]:
```suggestion
- Use **task pane add-ins** to bring in reference information or insert data into the presentation via a service. For example, see the [Pixton Comic Characters](https://appsource.microsoft.com/product/office/WA104380907) add-in, which you can use to add professional photos to your presentation. 
```

Comment (kbrandl) [reply docs/powerpoint/powerpoint-add-ins.md]: Good suggestion; will do.

File: docs/powerpoint/powerpoint-add-ins.md (hunk 2 above)
Comment (kbrandl) [reply docs/powerpoint/powerpoint-add-ins.md]: Good suggestion; will do.

File: docs/powerpoint/powerpoint-add-ins.md (hunk 1 above)
Comment (kbrandl) [reply docs/powerpoint/powerpoint-add-ins.md]: Good suggestion; will do.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a81d9dc](https://github.com/OfficeDev/office-js-docs-pr/pull/894/commits/a81d9dcdbc7d1f966777f0e170f7d889fdc68a45)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-debugging.md) | :bulb:Suggestion | | [Details](#user-content-fc1f0b9d-ef4680f1)
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-json-autogeneration.md) | :bulb:Suggestion | | [Details](#user-content-32630023-ef4680f1)
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |

<a id="fc1f0b9d-ef4680f1"></a>
### [docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-debugging.md)
 - **Line 24**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Use-the-command-line-tools-to-debug&quot;&gt;command line instructions&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named Use-the-command-line-tools-to-debug.
 - **Line 35**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#debug-in-excel-online-by-using-the-browser-developer-tools&quot;&gt;using the browser tools to debug Excel Online&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named debug-in-excel-online-by-using-the-browser-developer-tools.
 - **Line 142**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Sideload-your-add-in&quot;&gt;Sideload your add-in&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-debugging.md doesn&#39;t contain a bookmark named Sideload-your-add-in.

<a id="32630023-ef4680f1"></a>
### [docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-json-autogeneration.md)
 - **Line 14**: **[Suggestion]** Illegal link: `&lt;a href=&quot;#Types&quot;&gt;Types&lt;/a&gt;` -- missing bookmark. The file excel/custom-functions-json-autogeneration.md doesn&#39;t contain a bookmark named Types.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904151639444439-894%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FVJsYgBYYdryBQCsVd8fdkvBfSjQbyUYxdfFCIakGLs%3D&st=2019-04-15T16%3A36%3A02Z&se=2019-05-16T16%3A41%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6cd908a](https://github.com/OfficeDev/office-js-docs-pr/pull/894/commits/6cd908aa61ecf50bcbb1f1fa9edb1098d20dffa9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-debugging.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-debugging.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-json-autogeneration.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/excel/custom-functions-json-autogeneration.md) | :white_check_mark:Succeeded | |
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |
[docs/quickstarts/excel-custom-functions-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-CATS-issues-April-2019/docs/quickstarts/excel-custom-functions-quickstart.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904151707327802-894%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xD1SE%2FbcFJ1pD0YubmBmaLHjhgnwRQzR6zmvDje3bsM%3D&st=2019-04-15T17%3A03%3A56Z&se=2019-05-16T17%3A08%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**